cluster-bootstrap/multicluster-observability/base/custom-alerts/aap-alerts-policy.yaml

```diff
@@ -40,13 +40,37 @@ spec:
                     groups:
                     - name: AnsibleAutomationPlatform
                       rules:
+                      - alert: AAPMetricMissing
+                        annotations:
+                          description: AAP metric missing, the number of samples should be equal to 44, the current value is {{`{{$value}}`}}
+                        expr: scrape_samples_scraped{job="automation-controller-service",namespace="ansible-automation-platform"} < 44
```

Comment:
this means that if AAP expose more metrics in a new version, we should update this number right?

Reply:
yes, also need to update our custom metric list.

Reply:
if the new version AAP exposes more metrics, we do not update the custom metric list, then we still collect the 44 metrics from AAP.

Reply:
but in the custom metrics list we don't have 44 whitelisted metrics, how do you get to 44? whitelisted + base ones?

Reply:
I tested in our dev cluster and get 44 AAP metrics.

Reply:
I think Prom will add some non-AAP metrics in AAP metric endpoint.